pkg/systemd/services/timer-dialog.jsx

```diff
@@ -154,58 +265,77 @@ const CreateTimerDialogBody = ({ owner }) => {
-                               validated={submitted && validationFailed.description ? "error" : "default"}
+                               validated={validationFailed.description ? "error" : "default"}
+                               onBlur={validate}
                                onChange={setDescription} />
                 </FormGroup>
                 <FormGroup label={_("Command")}
+                           isRequired
                            fieldId="command"
-                           validated={submitted && validationFailed.command ? "error" : "default"}
+                           validated={validationFailed.command ? "error" : "default"}
                            helperTextInvalid={commandNotFound ? _("Command not found") : _("This field cannot be empty")}>
                     <TextInput id='command'
+                               isRequired
                                value={command}
-                               validated={submitted && validationFailed.command ? "error" : "default"}
+                               validated={validationFailed.command ? "error" : "default"}
+                               onBlur={validate}
                                onChange={str => { setCommandNotFound(false); setCommand(str) }} />
                 </FormGroup>
                 <FormGroup label={_("Trigger")} hasNoPaddingTop>
                     <Flex>
                         <Radio value="specific-time"
                                id="specific-time"
                                name="boot-or-specific-time"
-                               onChange={() => setDelay("specific-time")}
+                               onChange={() => {
+                                   setDelay("specific-time");
+                                   formCheck();
+                               }}
                                isChecked={delay == "specific-time"}
                                label={_("At specific time")} />
                         <Radio value="system-boot"
                                id="system-boot"
                                name="boot-or-specific-time"
-                               onChange={() => setDelay("system-boot")}
+                               onChange={() => {
+                                   setDelay("system-boot");
+                                   formCheck();
+                               }}
                                isChecked={delay == "system-boot"}
                                label={_("After system boot")} />
                     </Flex>
                     { delay == "system-boot" &&
                     <FormGroup className="delay-group"
+                               isRequired
                                label={_("Delay")}
-                               validated={submitted && validationFailed.delayNumber ? "error" : "default"}
+                               validated={validationFailed['delay-number'] ? "error" : "default"}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18148-20230110-092821-56aa70de-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)